addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20593,3 +20580,16 @@ msgstr ""
 msgctxt "#39024"
 msgid "Dependencies"
 msgstr ""
+
+#empty strings from id 39025 to 39099
+
+#: system/settings/settings.xml
```

Comment:
mind adding a contextual help string here?
`#. Label of a setting to specify an action that should be triggered on startup of Kodi`